### ISSUE TITLE
Improper Tap Targets not detected

### ISSUE THREAD

**Issue by pacifier007** [GoogleChrome/lighthouse#12994]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://demo.tagdiv.com/newspaper_pro/

### What happened?

There are many small tap targets in post meta such as Author name that's too close to post titles. For example: 

> "Top Fashion Trends to Look for in Every Important Collection"
> Armin Vans

Armin Vans in the example above is too close to title but never detected. 

### What did you expect?

All these post meta tap targets are improperly sized and should be detected as such. Right now, lighthouse is reporting 100% correctly sized tap targets while that's not true.

Considering this the most popular WordPress news theme used on hundreds of thousands of websites, the incorrect detection should be hurting UX. 

### What have you tried?

I have tried it both at web.dev/measure and on my local Chrome installs. The same issue is present on other sites with  similarly overlapping targets, but they only get flagged on some type of HTML structure for some reason. 

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

7.5 and 8.0

### Chrome Version

92

**Comment by patrickhulce** [comment]
Thanks for filing @pacifier007!

This appears to be working as intended from what I can see. Are you referring to these author titles?

![image](https://user-images.githubusercontent.com/2301202/131366649-6494d88f-8151-4b65-bc4c-98258f4b933c.png)

The tap target doesn't need to be very large, so long as the overlap with *other* possible tap targets is small to have a large hitbox. As you can see from highlighted red square (which is what Lighthouse is using to simulate a finger), the overlap with the headline is minimal and below the failure threshold of 25%.

**Comment by pacifier007** [comment]
This is rather confusing. So the 48x48px guidelines is an arbitrary number which doesn't really mean anything? Perhaps clear guidelines on this in docs would be helpful.

Is there any tool I can use to simulate this red triangle overlap you have mentioned. It seems pretty arbitrary otherwise. 

Can someone tell me why is this being flagged? 
URL: https://output.jsbin.com/nufevasago/1 
- It has more margin below.
- The line-height is higher than originally shared example.
- Basically, there's more spacing between the links than the original shared. Then why is being flagged and the original isn't.

**Comment by patrickhulce** [comment]
I haven't looked deeply into it but on initial inspection the bounding box for the target itself is slightly smaller (14px vs 15px high) and it's *just* on the border of failing at 27% (threshold is 25%)

**Comment by pacifier007** [comment]
Thanks for the pointer. That does make sense the target is indeed small and should be taller. Unfortunately, making it even 18px didn't work. It's still detected as having an overlap with the title. Tested here: https://output.jsbin.com/joqavaruji/1

<img width="744" alt="Screen Shot 2021-08-31 at 12 11 57 AM" src="https://user-images.githubusercontent.com/34238882/131392294-aba0a5fd-9eda-4b50-9e43-4ac351487d81.png">

I am confused since it has much more bottom margin and line-height than original example.

**Comment by patrickhulce** [comment]
@pacifier007 that URL also moved the upper headline though. For example, if I just increase the font-size to 14px of the "Armin Vans" without other adjustments, it starts passing.

You have made your point though that the original one should be failing if these are just barely failing, and I suspect the real issue is elsewhere that *something* excludes the node from consideration on the original URL you provided. We'll need to look into it, but there are many reasons this happens *on purpose* (time savings, false positive avoidance, etc), so what we find might not lead to any fix.

**Comment by connorjclark** [comment]
we replaced our tap-targets audit with axe's https://github.com/GoogleChrome/lighthouse/pull/15906
